internal/cmd/server.go

```diff
@@ -80,6 +80,7 @@ func newServerCmd() *cobra.Command {
 	cmd.Flags().Duration("session-inactivity-timeout", 0, "A user must interact with Infra at least once within this amount of time for their session to remain valid")
 	cmd.Flags().Bool("enable-signup", false, "Enable one-time admin signup")
 	cmd.Flags().String("base-domain", "", "base-domain for the server, eg example.com")
+	cmd.Flags().String("login-domain-prefix", "", "The path prefix on the base-domain that clients are redirected to after social login")
```

Comment:
Does this need to be a CLI flag? There's some other options in here that are probably unnecessary as flags

Reply:
Definitely would be nice to avoid adding new flags. I think most people will be configuring the server with either config files or env vars. Command line flags tend to be for interactive commands expected to be run by a human.

Reply:
Lets leave it here for now for consistency, all the other server options are available here. It's nice to be able to run the server locally without setting an environment variable.